airflow/providers/postgres/hooks/postgres.py

```diff
@@ -362,3 +362,12 @@ def get_openlineage_database_dialect(self, connection) -> str:
     def get_openlineage_default_schema(self) -> str | None:
         """Returns current schema. This is usually changed with ``SEARCH_PATH`` parameter."""
         return self.get_first("SELECT CURRENT_SCHEMA;")[0]
+
+    @staticmethod
+    def get_ui_field_behaviour() -> dict[str, Any]:
+        return {
+            "hidden_fields": [],
+            "relabeling": {
+                "schema": "Database",
+            }
```

Comment:
Static checks

```suggestion
            },
```

Reply:
Sorry, my bad. Missed this. Should've run the pre-commit checks before sending in.